.gitignore

```diff
@@ -11,3 +11,9 @@ cert.crt
 server-package.json
 .idea/httpRequests/
 data/
+*.log
+pnpm-lock.yaml
+package.json
+.gitignore
+.gitignore
+package.json
```

Comment:
Most of these changes seem either irrelevant (*.log) or wrong (package.json, .gitignore).